frappe/public/less/quill.less

```diff
@@ -131,13 +131,28 @@
 
 .ql-editor ol {
 	padding-left: 2.5em;
+  padding-right: 2.5em;
 }
 
 .ql-editor li[data-list=ordered] {
 	list-style-type: decimal;
 	padding-left: 0;
+  padding-right: 0;
 
 	&::before {
 		content: none;
 	}
 }
+
+// RTL alignment
+.control-input {
+  direction: ltr;
+}
+
+.ql-container {
+  direction: ltr;
+}
+
+.page-body {
+  overflow-x: visible;
+}
```

Comment:
Use tabs instead of spaces to indent.
```suggestion

// RTL alignment
.control-input {
	direction: ltr;
}

.ql-container {
	direction: ltr;
}

.page-body {
	overflow-x: visible;
}
```